sktime/forecasting/base/adapters/_fbprophet.py

```diff
@@ -43,7 +44,7 @@ def _fit(self, y, X=None, fh=None, **fit_params):
         """
         self._instantiate_model()
         self._check_changepoints()
-        self._set_y_X(y, X, enforce_index_type=pd.DatetimeIndex)
+        y, X = check_y_X(y, X, enforce_index_type=pd.DatetimeIndex)
```

Comment:
if you really insist to make this change to the fbprophet adapter, it should be controlled by the tag `enforce-index-type`

Reply:
I did this in order deprecate the `_set_y_X` as it was only used there and there is basically no additional functionality. Aso the index type was already enforced. But I can add a tag there, yes.